app/models/concerns/nested_ancestry_common.rb

```diff
@@ -7,8 +7,8 @@ module NestedAncestryCommon
     has_ancestry :orphan_strategy => :restrict
 
     before_validation :set_title
-    after_save :set_other_titles, :on => [:update, :destroy]
-    after_save :update_matchers, :on => :update, :if => proc { |obj| obj.saved_change_to_title? }
+    after_save :set_other_titles
+    after_save :update_matchers, :if => proc { |obj| obj.saved_change_to_title? }
```

Comment:
This looks like changed logic, should these be `after_update`?
Also, looks like @mmoll already did this change (plus another place) in https://github.com/theforeman/foreman/pull/7322 - let's spin it off to a separate pr cleaning up all the `:on`s and merge that.

Reply:
Extracted this to #7566

Reply:
Awesome, thanks :heart: